samples/fabric/simplestatewithacl/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.15
 require (
 	github.com/golang/protobuf v1.5.2
 	github.com/hyperledger-labs/weaver-dlt-interoperability/core/network/fabric-interop-cc/libs/testutils v0.0.0-20210909191523-de832057a3ab
-	github.com/hyperledger-labs/weaver-dlt-interoperability/core/network/fabric-interop-cc/libs/utils v1.3.1
+	github.com/hyperledger-labs/weaver-dlt-interoperability/core/network/fabric-interop-cc/libs/utils v1.5.6
```

Comment:
won't you also need to update go.sum as well?
there's a tool go-checksum that can be used, or also you can do that in separate PR after modules are published.

Reply:
But the `go.sum` won't even be generated unless the `1.5.6` version of `libs/utils` gets published, no?

Reply:
yeah it won't be generated automatically, you can do it in next PR then, when you enable the workflows